front/app/containers/Admin/dashboard/components/TimeControl.tsx

```diff
@@ -57,6 +57,7 @@ interface Props {
   startAtMoment?: Moment | null;
   endAtMoment: Moment | null;
   onChange: (startAtMoment: Moment | null, endAtMoment: Moment | null) => void;
+  hidePresets?: boolean;
```

Comment:
Instead of adding a `hidePresets` prop to hide everything except the `DateRangePicker`, is it possible to just import the `DateRangePicker` directly in the `AnalyticsChartWidget`?

Reply:
Yeah, totally should have done that. Done.